android-components/components/feature/search/src/main/assets/search/list.json

```diff
@@ -856,7 +856,7 @@
     "uk": {
       "default": {
         "visibleDefaultEngines": [
-          "google-b-m", "ddg", "wikipedia-uk"
+          "google-b-m", "bing", "ddg", "wikipedia-uk"
```

Comment:
I didn't see this in https://bugzilla.mozilla.org/show_bug.cgi?id=1818321#c1. Should we be adding this?

Reply:
Yes, it's in spreadsheet if you see. I forgot to add that to the bug comment. I can do it.